### ISSUE TITLE
"Not tested with the active version..." on ALL WooCommerce Plugins since 3.4.6

### ISSUE THREAD

**Issue by oliverdavidkrieg** [woocommerce/woocommerce#21618]
Hi everyone,

I updated to 3.4.6 last week and now to 3.4.7 but I seem to have an alarming amount of plugins that WooCommerce describes as not yet tested in terms of compatibility.

What makes it alarming is that even WooCommerce's own additional plugins tell me they are not yet tested. Is that normal? I get this notification about WooCommerce Germanized, but when I go into WooCommerce Status Report I can see the following pluings are "not yet tested" :

Mailchimp for WooCommerce
WooCommerce Variation Swatches
WooCommerce Currency Switcher
WooCommerce PayPal Checkout Gateway
WooCommerce Stripe Gateway
WooCommerce Germanized Pro
WooCommerce Germanized
WooCommerce Multilingual 
WooCommerce Services

This must be a bug?

**Comment by thomas-zg** [comment]
This happens and not worked anymore since it was introduced a few versions ago.

**Comment by oliverdavidkrieg** [comment]
Okay, as long as I don't have to do anything to fix it I'm happy. I don't like having red error messages on my dashboard but, well, it's not that bad if it's actually still working fine.

**Comment by thomas-zg** [comment]
Gerhard,

This is the screen in the plugin's folder:
![grafik](https://user-images.githubusercontent.com/25877146/47650268-0c180e00-db80-11e8-8f1f-33a64be87071.png)

and this is the header of one of the listet plugins:
/**
 * Plugin Name: WooCommerce PDF Invoices & Packing Slips
 * Plugin URI: http://www.wpovernight.com
 * Description: Create, print & email PDF invoices & packing slips for WooCommerce orders.
 * Version: 2.2.3
 * Author: Ewout Fernhout
 * Author URI: http://www.wpovernight.com
 * License: GPLv2 or later
 * License URI: http://www.opensource.org/licenses/gpl-license.php
 * Text Domain: woocommerce-pdf-invoices-packing-slips
 * WC requires at least: 2.2.0
 * WC tested up to: 3.5.0
 */

Or the Stripe Gateway and the Status report:
![grafik](https://user-images.githubusercontent.com/25877146/47650676-43d38580-db81-11e8-94ea-ae3aa7854dbb.png)


<?php
/**
 * Plugin Name: WooCommerce Stripe Gateway
 * Plugin URI: https://wordpress.org/plugins/woocommerce-gateway-stripe/
 * Description: Take credit card payments on your store using Stripe.
 * Author: WooCommerce
 * Author URI: https://woocommerce.com/
 * Version: 4.1.12
 * Requires at least: 4.4
 * Tested up to: 4.9
 * WC requires at least: 2.6
 * WC tested up to: 3.5
 * Text Domain: woocommerce-gateway-stripe
 * Domain Path: /languages
 *
 */

**Comment by oliverdavidkrieg** [comment]
it's the same problem for me, although after the last round of updates the error is only occuring for WooCommerce Variation Swatches, Currency Switcher, and Mailchimp.

I confirm, everything works fine.

**Comment by kloon** [comment]
OK thanks, will reopen se we can take a look at this further.

**Comment by kloon** [comment]
I did a bunch of tests now and cannot replicate this, it is possible that this was fixed with #21669.

Wait for 3.5.2 and if the issue persists then we can reopen again, but seeing as I am unable to replicate I do not see this as a bug anymore.

Just a note that you will get notices about WooCommerce that has not declared a tested up to version, we just do that to make people aware that the developer did not specifically declare support in their plugin for the version of WooCommerce you are upgrading to.
